ui/src/components/DataTable/index.jsx

```diff
@@ -85,11 +85,12 @@ export default class DataTable extends Component {
       sortByHeader,
       sortDirection,
       noItemsMessage,
+      ...props
```

Comment:
There's been some changes to this file. I think a rebase is required.

Reply:
So that's exactly why I was insisting on merging this to master